traits/trait_base.py

```diff
@@ -187,15 +187,15 @@ def enum_default(values):
 
     Parameters
     ----------
-    values : tuple, list or enum.Enum
+    values : tuple, list, collection or enum.Enum
```

Comment:
Couldn't this simply be `values : collection`, since all the other listed cases are special cases of collections?

Reply:
Function removed